packages/amplify_auth_plugin_interface/lib/src/Attribute/FetchUserAttributesOptions.dart

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@Deprecated("Please use 'FetchUserAttributesOptions' instead")
+class AuthUserAttributeOptions extends FetchUserAttributesOptions {}
```

Comment:
*Note:* This class was renamed for consistency. It is part of the public API. There is no Cognito implementation/extension of it. The only purpose it serves right now is to allow a customer that is creating a custom auth plugin (instead of using Cognito auth) to extend it to allow for additional options in their implementation of this fetchUserAttributes.

Dart 2.12 and lower doesn't have type aliasing, so this seemed to be the most straightforward way to deprecate it.